### PR TITLE
ARC-413 readable logs

### PR DESCRIPTION
**Pull request by rachellerathbone**
Minor update to the logger so that logs are human-readable when locally but still print in JSON format for all Micros envs.

### CODE REVIEW

File: src/config/logger.ts [hunk 1]
```diff
@@ -2,8 +2,12 @@ import Logger, {levelFromName} from 'bunyan';
 import bformat from 'bunyan-format';
 import filteringStream from '../util/filteringStream'
 
+// For any Micros env we want the logs to be in JSON format.
+// Otherwise, if local development, we want human readable logs.
+const outputMode = process.env.MICROS_ENV ? 'json' : 'short';
+
 // add levelInString to include DEBUG | ERROR | INFO | WARN
-const formatOut = filteringStream(bformat({outputMode: 'bunyan', levelInString: true}));
+const formatOut = filteringStream(bformat({ outputMode, levelInString: true }));
```
Comment (KAbakumovAtl) [inline src/config/logger.ts:10]: I wonder what's the difference between 'json'  and 'bunyan' here? Did you test it in micros?

**Comment by KAbakumovAtl** [comment]
That's a great idea! Happy with the change as long as logs have the same format in Splunk

File: src/config/logger.ts (hunk 1 above)
Comment (mboudreau) [reply src/config/logger.ts:10]: Looking at the code for bunyan-format, they're the same except for `json` which has indentation (multiple line) vs bunyan which is all one line.  Literally `JSON.stringify(log)` vs `JSON.stringify(log, null, 2)`.

**Review by mboudreau — APPROVED** [review]
yay!

**Pull request merged**